### DISCUSSION TITLE
On compatibility of the Veikk A15 Pro, relative mode (similar to mouse mode) and scroll dial support.

### DISCUSSION THREAD

**Discussion by andreluizrcn** [kurikaesu/userspace-tablet-driver-daemon#74] [General]
Hello! hope this message meets you well!! I'm familiar on using the Open Tablet Driver software, but it doesn´t have supports dial funcion (scroll wheel) yet. I would like to know if your project can recognize my tablet and it's functions keys, including the dial wheel (similar to other products on wacom or huion that has this dial function key).
Thanks for your time and work!  :)

**Comment by kurikaesu** [comment]
Hello there, thanks for reaching out.

Out of the box, my project will most likely not support the tablet that you are using. While it might not be too big an effort to add support for your tablet in, it would require someone writing the code for it. I unfortunately don't own this tablet and I have already too many as it is to purchase another one.

If you want to try implementing it yourself, I would be more than happy to assist

**Reply by andreluizrcn** [reply]
I see :( , thanks so much!!!!!!!!!!!!!!!
